addons/asset-conflict-dialog/style.css

```
@@ -0,0 +1,33 @@
+.conflictDialog {
+  box-sizing: border-box;
+  width: 600px;
+  max-height: 80vh;
+  max-width: 85%;
+  margin-top: 12vh;
+  overflow-y: auto;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.conflictDialog .sa-editor-modal-content {
+  padding: 1.5rem 2.25rem;
+}
+
+.conflictDialog .sa-editor-modal-content > p {
+  margin-top: 0;
+}
+
+.conflictDialog-actions {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 1rem;
+}
+
+.conflictDialog-actions > button:hover {
+  background-color: hsla(260, 60%, 60%, 0.15);
+}
+
+.conflictDialog-footer {
+  border-top: solid 1px #aaa;
```

```suggestion
  border-top: solid 1px var(--editorDarkMode-border, rgba(0, 0, 0, 0.15));
```
For consistency with other borders and dark mode support.